core/src/cats/ddl/creates/postgresql.sql

```diff
@@ -171,7 +171,7 @@ CREATE TABLE Media
     VolWrites         BIGINT      DEFAULT 0,
     VolCapacityBytes  BIGINT      DEFAULT 0,
     VolStatus         TEXT        NOT NULL
-        CHECK (VolStatus IN ('Full', 'Archive', 'Append',
```

Comment:
we need migrations to do that on upgrade and the schema version needs to be updated when schema changes.